src/libxively/xi_debug.c

```diff
@@ -37,3 +38,29 @@ void xi_debug_data_logger_impl( const char* msg, const xi_data_desc_t* data_desc
 
     xi_free( tmp );
 }
+
+/** Trims the absolute file path to just the file name */
+const char* xi_debug_dont_print_the_path( const char* msg )
```

Comment:
I was wondering about compile time solutions. Searched for another macro but there is not __ONLY_FILENAME__ or such. But they claim
gcc file.c
gcc path/to/source/file/file.c
http://stackoverflow.com/questions/8487986/file-macro-shows-full-path
sets the __FILE__ macro. We cannot use this but
http://stackoverflow.com/questions/7398310/how-to-get-only-file-name-in-preprocessor
says a compiler flag defined as this:
-DXI_BASE_FILENAME=\"$(<F)\"
could provide you the only the filename. Which you could use in the print functions instead of __FILE__

What about this compile time solution vs the runtime path parsing?